tensorboard/webapp/notification_center/_views/notification_center_test.ts

```diff
@@ -137,8 +138,88 @@ describe('notification center', () => {
       store.overrideSelector(selectors.getLastReadTime, 1);
       const fixture = TestBed.createComponent(NotificationCenterContainer);
       fixture.detectChanges();
+
       const reddot = fixture.debugElement.query(By.css('.red-dot'));
       expect(reddot).toBeNull();
     });
   });
+  describe('full note link', () => {
+    it('appears when the link is provided', () => {
+      store.overrideSelector(selectors.getNotifications, [
+        {
+          category: CategoryEnum.WHATS_NEW,
+          dateInMs: 1,
+          title: 'test title',
+          content: 'test content',
+          fullNoteLink: 'https://google.com',
+        },
+      ]);
+      const fixture = TestBed.createComponent(NotificationCenterContainer);
+      fixture.detectChanges();
+      const menuButton = fixture.debugElement.query(
+        By.css('[aria-label="Display notification messages"]')
+      );
+      menuButton.nativeElement.click();
+      fixture.detectChanges();
+
+      const linkElement = fixture.debugElement.query(
+        By.css('.extended-buttons a')
+      );
+
+      expect(linkElement).toBeTruthy();
+      expect(linkElement.nativeElement.href).toEqual('http://google.com');
```

Comment:
`fullNoteLink` is now `'http**s**://google.com',` but we assert `http://`. Is that correct?

Reply:
Don't think this question is resolved. PTAL.